packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -75,19 +75,21 @@ The app logo that you submit to the OIN Manager in the **General Settings** > **
 
 ## App description guidelines
 
-The content in the OIN Manager **General Settings** > **App description** field is displayed on the Integration detail page under the **Overview** heading for your published integration. Descriptions must be less than 1024 characters and should describe the value that you provide to customers by partnering with Okta. See an app description example in this [Integration detail page](https://www.okta.com/integrations/servicenow-ud/).
+From the OIN Manager, you can specify the app description in the **General Settings** > **App description** field. Descriptions must be less than 1024 characters and should describe the value that you provide to customers by partnering with Okta.
+
+You can find the app description in the OIN catalog under the **Integration detail** > **Overview** section of your published integration. See an app description example in this [Integration detail page](https://www.okta.com/integrations/servicenow-ud/).
```

Comment:
I see Ops team very commonly sending ISVs examples of descriptions, can we add these examples? 

Here are some examples:
Infinipoint enables enterprises of all sizes to automate cyber hygiene across their entire IT estate with unprecedented speed and scale. For the benefit of single-sign-on, Okta's Infinipoint integration gives your team the ability to securely manage and secure the organisation's IT assets.
Five to Nine is a company event discovery, management and evaluation software. From single sign-on to user provisioning, Okta’s Five to Nine integration handles user access throughout the user’s lifecycle from onboarding to offboarding. The integration also allows you to import custom user attributes from Okta into Five to Nine.
Enables Edify customers to log into the Edify platform using Okta as a single sign-on provider`

These are what the Ops team email to ISVs as example  good descriptions.

Reply:
I talked with Mohammad Abdullah: he gave me a generic description to use. We don't have approval to use Infinipoint or Five to Nine descriptions.